packages/edit-site/src/components/preferences-modal/enable-feature.js

```diff
@@ -6,16 +6,21 @@ import { ___unstablePreferencesModalBaseOption as BaseOption } from '@wordpress/
 import { store as preferencesStore } from '@wordpress/preferences';
 
 export default function EnableFeature( props ) {
-	const { featureName, onToggle = () => {}, ...remainingProps } = props;
+	const {
+		namespace = 'core/edit-site',
```

Comment:
The prop could be called `scope`.

But also I wonder if there could be one component in the editor package for this that has 'core/edit-post' hardcoded. Any migrated preferences could use the editor package component, then eventually the edit-site and edit-post ones could be deprecated/removed.

Reply:
Yes, I think we need to unify the UI down the road which would mean updating this component a bit. I think for now it's fine to make this change in the component. I'll update to use scope.